mitmproxy/filt.py

```diff
@@ -155,9 +155,9 @@ class FHead(_Rex):
     flags = re.MULTILINE
 
     def __call__(self, f):
-        if f.request and self.re.search(bytes(f.request.headers)):
+        if f.request and self.re.search(bytes(f.request.headers).decode()):
```

Comment:
mitmproxy.filt is unfortunately not as trivial to port as this may look.
I propose we open a separate issue for that to not clog up things here. :smiley: